examples/tracegen/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  # Jaeger
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - "16686:16686"
+      - "14268"
```

Comment:
This can be removed when using the Jaeger exporter with gRPC (14250).

Reply:
fixed.

Reply:
fixed.